### ISSUE TITLE
Unidentified lag. 

### ISSUE THREAD

**Issue by Limuruae** [Wynncraft/Issues#6929]
Alright, to start it off, I got a new computer in February, and everything worked fine for about a week and once 7-8 days passed by, I started getting quite high ping lag. I figured, since a lot of games were coming out I'd just let Minecraft work it out by itself, so I took a break for a few months and came back around April, still ping lag. And what I mean by ping lag is it running on 50-80ms/s and then it fluctuating between 500+. So I called up my ISP (Cox Communications) and they delivered me a new Ethernet cable, and a new modem. And it didn't fix the problem, so I decided to try one more last time to wait it out. Fast forward to 2-3 days ago and I decided I had enough, called up my ISP and told them to send a guy out and to figure this out. He told me, and I quote, "Nothing's wrong with your computer, modem or anything of the such. The servers on Minecraft are capping you at a certain speed so you get lag spikes..." And I really doubt that, but you get the point. I factory-reset my computer, turned WiFi off, bought a new PC Networking chip, everything. I want to so badly get this fixed.. Wynncraft's my favorite server. :/

Specs: Intel iCore i5-4460 CPU @ 3.20 Ghz RAM: 8.00 GB and my card is NVIDIA GTX 750

Speedtest: Ping-38ms DS- 109.01mbps UP-21.08mpbs

Pingtest:  65ms, Jitter 2ms

**Comment by ColinRioux** [comment]
Well, the bad news is, I am not getting high ping lag. I get between 40-45ms of ping, with a slower download and upload speed.  You have been doing the right things by calling up your ISP and so forth, and having them troubleshoot the best way they can. In fact, you're ahead of the game, so you've been extremely helpful. 

Although I am in no way a technician or network engineering, I have found some forum posts online that you may find helpful. 
[1](http://forums.cox.com/forum_home/internet_forum/f/5/t/10966.aspx)
[2](http://www.tomshardware.com/forum/28006-42-extremely-high-ping-help)
[3](https://us.battle.net/forums/en/wow/topic/3196392643) 
What I got from the third one is that Cox may be monitoring your network, and throttling it as a result. 

Although your results from Cox may seem that your computer is now the problem, I don't think you should eliminate Cox completely. ISPs are the same, and sometimes, will say everything is fine, while it's just not (because they are trying to save money). I've had my bursts with Comcast. 

Can you confirm this is only happening on Wynncraft? Have you tried other major networks to see if you come up with the same result? Have you tried other games? 

If you have no problem with other games, and/or another server, we could deduce your problem to your computer. Some other games take up way less network usage than minecraft.

**Comment by Limuruae** [comment]
I can confirm that this is happening with all of my Minecraft. Minecraft is the only game that I'm having ping lag issues with. I just spent the last 10 minutes going through 94+ games on steam/browser. Minecraft (and Trove) are the only games that my ping lag is happening with.

**Comment by ColinRioux** [comment]
Interesting, does this occur with all versions of minecraft? (I doubt the version matters, but want to make sure).

**Comment by Limuruae** [comment]
Yes, I just went to my local Cox store, and picked up a new modem and it's still doing it. I just turned my WiFi off so it doesn't use both wired & WiFi. I'm about to just make them get a tech out here and re-do all of the wiring.. I just want to play my favorite server like I used to be able to :/

**Comment by Limuruae** [comment]
It's looking like it's my computer.. But what would cause it?

**Comment by ColinRioux** [comment]
It could very well be the communication between your computer and your router. 

Is there absolutely nothing different between your current computer and your old one?

One last thing, that I can think of currently, is adding a DNS server or two. Probably google DNS or openDNS is the best option. See if that does anything.

**Comment by Limuruae** [comment]
The only things that are different are that my new computer is better in every single way, (SPECS wise), and that it's Windows 10 instead of 8. I just plugged in my old computer, and ran it on Wynn, guess what? Perfect connection (besides the 20-40 fps).

**Comment by Limuruae** [comment]
Just tried to downgrade from Windows 10, no success.

**Comment by ColinRioux** [comment]
Considering this is only occurring when you are playing minecraft and trove, right now, I can't seem to troubleshoot what is going on. Certainly I could figure something out soon. I'm sorry.

**Comment by Limuruae** [comment]
It's fine! I think I'm about to give up on playing Minecraft for good, thanks for trying!

**Comment by Moe-Ronickah** [comment]
Best guess - your firewall settings need tweaking.  Look up settings for MC in Win 10, and see what you come up with.
